### PR TITLE
Clarify the Upgrade Documentation

### PR DESCRIPTION
**Pull request by blueandgold**

### CODE REVIEW

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 1]
```diff
@@ -8,13 +8,16 @@ Note: If you used the Forseti setup wizard to deploy, your `deploy-forseti.yaml`
...
 ### Get a particular deployment template version
-This step is **optional**.
+-This step is **optional**.
 
 If you need to change the `release-version` for your Forseti deployment, you MUST 
 get the correct version of Forseti. The deployment template's startup 
 script has release-specific code, so things will break if you use a startup script that 
 is out of sync with the deployed release.
 
+You need to be where you previously retrieved the code from GitHub
```
Comment (Tabrarian) [inline _docs/howto/deploy/change-gcp-deployment.md]: What does this bit mean? I have to use the same tool and I have to be in the same directory?

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 2]
```diff
@@ -50,26 +53,44 @@ is out of sync with the deployed release.
...
+properties have changed.
+
+   ```yaml
+   region: $(ref.cloudsql-instance.region)
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+So, you would have to copy these new properties over to your generated
```
Comment (Tabrarian) [inline _docs/howto/deploy/change-gcp-deployment.md]: So, you would have to copy these new properties over to your generated
+`deploy-forseti-<timestamp>.yaml` and update the placeholders to the actual 
+values that you will use, e.g. the project id of the project that Forseti is 
+running in, "default", "default".

To upgrade, copy these new properties to your generated `deploy-forseti-<timestamp>.yaml`. Then, update the placeholders to the values you want to use. For example, the project id of the project that Forseti is running in, "default", "default".

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 1 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: This is to say that users have to be at the local repo where they had run their previous deployment.  It does seem not needed, so I'll drop it.

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 2 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: Done!

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 3]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+
+To upgrade, copy these new properties to your generated 
+deploy-forseti-<timestamp>.yaml. Then, update the placeholders to the values 
+you want to use. For example, the project id of the project that Forseti is 
+running in, "default", "default".
+
+1. Inspect `deploy-forseti-<timestamp>.yaml` and verify if your ```branch-name``` 
+   property is hardcoded to a specific version.  If so, update it to the latest
```
Comment (Tabrarian) [inline _docs/howto/deploy/change-gcp-deployment.md]: Remove the extra space: "... version. If so..."

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 4]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
-   $ git diff origin..HEAD~1 -- deploy-forseti.yaml.sample
+   $ git diff origin/master..HEAD~1 -- deploy-forseti.yaml.sample
    ```
 
-1. Edit `deploy-forseti.yaml` and update the values you want to change. If you 
-   previously deployed the `master` branch, you don't need to change it.
+1. Edit `deploy-forseti-<timestamp>.yaml` and update the values of the new properties.
+
+For example, from v1.1.7 to v1.1.10, the following Compute Engine instance 
+properties have changed.
```
Comment (mirons-google) [inline _docs/howto/deploy/change-gcp-deployment.md]: have been changed and/or added.

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 5]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
+   ```yaml
+   region: $(ref.cloudsql-instance.region)
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+
+To upgrade, copy these new properties to your generated 
+deploy-forseti-<timestamp>.yaml. Then, update the placeholders to the values
```
Comment (mirons-google) [inline _docs/howto/deploy/change-gcp-deployment.md]:
```
`deploy-forseti-<TIMESTAMP>.yaml`
```

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 6]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
+   region: $(ref.cloudsql-instance.region)
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+
+To upgrade, copy these new properties to your generated 
+deploy-forseti-<timestamp>.yaml. Then, update the placeholders to the values 
+you want to use. For example, the project id of the project that Forseti is
```
Comment (mirons-google) [inline _docs/howto/deploy/change-gcp-deployment.md]: ... the id of the project that...

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 7]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
+
+   network-host-project-id: NETWORK_HOST_PROJECT_ID
+   vpc-name: VPC_NAME
+   subnetwork-name: SUBNETWORK_NAME
+   ```
+
+To upgrade, copy these new properties to your generated 
+deploy-forseti-<timestamp>.yaml. Then, update the placeholders to the values 
+you want to use. For example, the project id of the project that Forseti is 
+running in, "default", "default".
```
Comment (mirons-google) [inline _docs/howto/deploy/change-gcp-deployment.md]: `, "default", "default".` is this intended?

File: _docs/howto/deploy/change-gcp-deployment.md [hunk 8]
```diff
@@ -50,26 +50,45 @@ is out of sync with the deployed release.
...
+
+1. Inspect `deploy-forseti-<timestamp>.yaml` and verify if your ```branch-name```
```
Comment (mirons-google) [inline _docs/howto/deploy/change-gcp-deployment.md]:
```
`deploy-forseti-<TIMESTAMP>.yaml`

Comment (mirons-google) [reply _docs/howto/deploy/change-gcp-deployment.md]: upper case TIMESTAMP everywhere please.

File: _docs/quickstarts/scanner/rules.md [hunk 9]
```diff
@@ -67,6 +69,8 @@ rules:
 
 ## Kubernetes Engine (KE) rules
```
Comment (Tabrarian) [inline _docs/quickstarts/scanner/rules.md]: Please don't abbreviate Kubernetes Engine when referring to the Google product. We should always use the full product name. Thanks!

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 4 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: Done

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 5 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: Done

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 6 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: Done

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 7 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: correct, "default" are the actual values

File: _docs/howto/deploy/change-gcp-deployment.md (hunk 8 above)
Comment (blueandgold) [reply _docs/howto/deploy/change-gcp-deployment.md]: Done

File: _docs/quickstarts/scanner/rules.md (hunk 9 above)
Comment (blueandgold) [reply _docs/quickstarts/scanner/rules.md]: Done

**Pull request merged**